lib/DefinePlugin.js

```diff
@@ -25,6 +32,7 @@ const stringifyObj = obj => {
 const toCode = code => {
 	if (code === null) return "null";
 	else if (code === undefined) return "undefined";
+	else if (code instanceof RuntimeValue) return toCode(code.fn());
```

Comment:
Best if you would move the file dependencies registration here. Try to pass the `parser` as argument to `toCode` (and `stringifyObj`).

Reply:
That makes sense.
Thanks for your patience with me as I try to figure this all out ;)